composites/SnippetPreview/components/ProgressBar.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import colors from "../../../style-guide/colors.json";
+
+const ProgressBarContainer = styled.progress`
+	box-sizing: border-box;
+	width: 100%;
+	height: 8px;
+	display: block;
+	margin-top: 5px;
+	border: none;
+	-webkit-appearance: none;
+	appearance: none;
+	
+	::-webkit-progress-bar {
+	   	background-color: ${colors.$color_background_light};
+	   	color: ${( { color } ) => color };
```

Comment:
Line 17 and 18 have spaces used for indentation, please use only tabs.
Line 17: please add spaces inside brackets